src/api/custom-resolvers/create-account.ts

```diff
@@ -20,6 +21,15 @@ const createAccount = async (root: any, variables: AccountCreationVariables, con
   try {
     const { firstName, lastName, email, password } = variables;
 
+    // check if an account with the email already exists
+    const exporter = await getAccountByField(context, 'email', email);
```

Comment:
can 'email' be a constant please

Reply:
Agreed will do this as part of next PR as there are a few instances of this